.github/workflows/pr.yaml

```diff
@@ -54,6 +55,24 @@ jobs:
       - name: Telemetry setup
         if: ${{ vars.TELEMETRY_ENABLED == 'true' }}
         uses: rapidsai/shared-actions/telemetry-dispatch-stash-base-env-vars@main
+  check-nightly-ci:
+    runs-on: ubuntu-latest
+    container:
+      # Just need an image where Python is available
```

Comment:
Could we use the default image then?

Reply:
From offline discussions, Bradley's suggestion is to use one of the images from https://github.com/actions/runner-images, specifically probably ubuntu-latest, so I'm switching to that.

Reply:
I had to use 24.04 to get a feature I need from Python 3.11. We can change to ubuntu-latest once that is 24.04 (or later).

Reply:
What feature? That's fine for now.

Reply:
https://docs.python.org/3/library/datetime.html#datetime.date.fromisoformat didn't support timezones before 3.11.